ui-packages/packages/runtime-tools-dev-ui-webapp/src/channel/CustomDashboardList/CustomDashboardListContext.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
```

Comment:
```suggestion
 * Copyright 2022 Red Hat, Inc. and/or its affiliates.
```

Please check all other new files that might have this typo as well.